modules/db-for-my-sql/flexible-server/main.bicep

```diff
@@ -26,6 +26,9 @@ param administratorLogin string
 @secure()
 param administratorLoginPassword string
 
+@description('Optional. The Azure AD administrators when AAD authentication enabled.')
+param administrators array = []
```

Comment:
Similar to the strongly related [DBforPostgreSQL implementation](https://github.com/Azure/ResourceModules/pull/3952/files#diff-64908f190540a7d5fb4a5d7389565068e4c8edb26659de29e7d57a98d7410566), shouldn't the original administrator inputs become optional with this feature?

Reply:
Very good catch. I aligned it to the Postgre one. Regenerated JSON and readmes